terraform/cloudfoundry/elbs.tf

```diff
@@ -93,6 +93,57 @@ resource "aws_proxy_protocol_policy" "cf_uaa_haproxy" {
   instance_ports = ["443"]
 }
 
+resource "aws_elb" "cf_system" {
```

Comment:
Can we name this something like `cf_router_system_domain` to make it clear that it's loadbalancing the routers.

Reply:
Sensible. Done.

Reply:
I had forgot to update the ELB name and bucket prefix, but that is now done, squashed, and force pushed.